common/djangoapps/student/management/commands/change_email_subscription_states.py

```diff
@@ -0,0 +1,68 @@
+"""Management command to change many user email subscription states at once."""
+import logging
+
+import unicodecsv
+from django.core.management.base import BaseCommand
+
+from lms.djangoapps.utils import get_braze_client
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+UNSUBSCRIBED = "unsubscribed"
+CHUNK_SIZE = 50
+
+
+class Command(BaseCommand):
+    """
+    Management command to change many user email subscription states at once.
+    """
+
+    help = """
+    Change the email subscription status for all given users on braze.
+
+    Example:
+
+    Change email subscription status for multiple users on braze.
+        $ ... change_email_subscription_states -p <csv_file_path>
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-p', '--csv_path',
+            metavar='csv_path',
+            dest='csv_path',
+            required=True,
+            help='Path to CSV file.')
+
+    def _chunked_iterable(self, iterable):
+        """
+        Yield successive CHUNK_SIZE sized chunks from iterable.
+        """
+        for i in range(0, len(iterable), CHUNK_SIZE):
+            yield iterable[i:i + CHUNK_SIZE]
+
+    def _chunk_list(self, emails_list):
+        """
+        Chunk a list into sub-lists of length CHUNK_SIZE.
+        """
+        return list(self._chunked_iterable(emails_list))
+
+    def handle(self, *args, **options):
+        csv_file_path = options['csv_path']
+
+        with open(csv_file_path, 'rb') as csv_file:
+            reader = list(unicodecsv.DictReader(csv_file))
+            emails = [row.get('email') for row in reader]
+
+        chunks = self._chunk_list(emails)
+
+        try:
+            braze_client = get_braze_client()
+            if braze_client:
+                for chunk in chunks:
+                    braze_client.change_email_subscription_state(
+                        email=chunk,
+                        subscription_state=UNSUBSCRIBED
```

Comment:
Can we pass this as a command argument? By using this parm we can subscribe users in the future.
Create a list of all available statuses in Braze like `unsubscribed ` `subscribed ` and verify user-entered status in command and then use this argument in the command.